src/Hosting/Hosting/src/HostingHostBuilderExtensions.cs

```diff
@@ -152,7 +152,7 @@ public static IHostBuilder UseConsoleLifetime(this IHostBuilder hostBuilder, Act
         /// Enables console support, builds and starts the host, and waits for Ctrl+C or SIGTERM to shut down.
         /// </summary>
         /// <param name="hostBuilder">The <see cref="IHostBuilder" /> to configure.</param>
-        /// <param name="cancellationToken"></param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> that can be used to cancel the health check.</param>
```

Comment:
Health check? I think this comment was erroneously copied in many many places. Please fix in all places.